packages/admin/admin/src/stack/breadcrumbs/BreadcrumbsOverflow.tsx

```diff
@@ -1,32 +1,45 @@
 import { ChevronRight } from "@comet/admin-icons";
 import { Link, ListItemIcon, ListItemText, Menu, MenuItem } from "@mui/material";
-import { WithStyles } from "@mui/styles";
-import clsx from "clsx";
+import { css, styled } from "@mui/material/styles";
 import * as React from "react";
 
 import { BreadcrumbItem } from "../Stack";
 import { BreadcrumbLink } from "./BreadcrumbLink";
-import { styles } from "./StackBreadcrumbs.styles";
 
-interface BreadcrumbsOverflowProps {
+export interface BreadcrumbsOverflowProps {
```

Comment:
There is no need to export this interface. 

```suggestion
interface BreadcrumbsOverflowProps {
```